pkg/instancetype/instancetype.go

```diff
@@ -399,6 +410,46 @@ func (m *methods) FindPreferenceSpec(vm *virtv1.VirtualMachine) (*instancetypev1
 	}
 }
 
+func (m *methods) FindPreferenceResourceAttributes(vm *virtv1.VirtualMachine, verb string) (*authv1.ResourceAttributes, error) {
```

Comment:
Given we don't need to actually look anything up here given @akrejcir other comments could you also rename this to `CreatePreferenceResourceAttributes` ?

Reply:
Done